org-code-javabuilder/lib/src/main/java/org/code/javabuilder/CodeBuilderWrapper.java

```diff
@@ -61,6 +61,9 @@ public void executeCodeBuilder() {
 
       // Additionally, these may have affected the user. For now, let's tell them about it.
       outputAdapter.sendMessage(error.getExceptionMessage());
+    } finally {
+      GlobalProtocol.getInstance().cleanUpResources();
```

Comment:
this is so that we send any messages if `start` is not called, or the program exits at an unexpected time. We want to show the initial state even if we don't call start, so we need to do this clean up once the program has exited.